src/css/components/Checkbox.scss

```diff
@@ -169,13 +169,16 @@ $Checkbox-bg-color: $cu-foreground !default;
   background-image: url("data:image/svg+xml; utf8, %3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 960 960'%3E%3Cpath fill='white' d='M915 178.9c8 6.7 12.3 15.5 13 26.5s-2.3 21.2-9 30.5c-252.7 360.7-389.3 555.3-410 584-21.3 28.7-48.8 43.2-82.5 43.5S365 849.2 343 819.9l-278-391c-6.7-9.3-9.7-19.7-9-31s5-20.3 13-27c34-29.3 72-55.7 114-79 8.7-4.7 18.3-5.3 29-2s19.3 9.7 26 19l188 264 321-456c6.7-9.3 15.2-15.5 25.5-18.5s20.2-2.2 29.5 2.5c44 24 81.7 50 113 78z'%3E%3C/path%3E%3C/svg%3E");
   // SVG check icon
   border: none;
+  transition: none
 }
 
 // Indeterminate state
 .Checkbox--indeterminate .Checkbox-input:checked + .Checkbox-label:before {
   background-image: url("data:image/svg+xml,%3Csvg width='4' height='4' viewBox='0 0 8 8' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='1.5' y='1.5' width='5' height='5' rx='1.5' fill='%230F52BA' stroke='%230F52BA'/%3E%3C/svg%3E%0A");
```

Comment:
Sorry should have called this out but I updated these attributes in the SVG code too:
`x='2' y='2' width='4' height='4' rx='1'`

Reply:
@saralohr sorry, which SVG? The Active one?

Reply:
Oh sorry, the indeterminate one. It just makes the inner box a little smaller since we are making the background size 100%.